recipes/geometric/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "geometric" %}
+{% set version = "0.8.7" %}
+
+# openssl sha256 filename
+{% set sha256 = "91421168d7c04305230a30dbaa96ba61832e6ff3dad49b07c686f60a5667b862" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: geomeTRIC-{{ version }}.tar.gz
+  url: https://github.com/leeping/geomeTRIC/archive/{{ version }}.tar.gz
```

Comment:
Since `package/name` applies `.lower()`, you can probably use `{% set name = "geomeTRIC" %}`, then use the templating in `source` fields.

But more importantly, yes, give `0.8.8` a try.

Reply:
`0.8.8` seems to have fixed the issue and I made the suggested changes.